public/ui.js

```diff
@@ -0,0 +1,7 @@
+/* Adds a red asterix next to required fields */
+$(function(){
```

Comment:
I feel as though this code should be in app/assets/javascripts/application.js or a relevant file at the same path like probable-engine/app/assets/javascripts/forms.js
That way rails renders it into the public folder through pre-compilation in production.